julia/test/evaluate.jl

```diff
@@ -17,10 +17,20 @@
     @testset "evaluate juliaize" begin
         @test juliaize(evaluate("1")) === Int32(1)
         @test juliaize(evaluate("1.5")) === 1.5f0
+        @test juliaize(evaluate("IToW8 65")) === UInt8(65)
+    end
 
-        @test juliaize(evaluate("IToW8 65")) === Int8(65)
+    @testset "juliaize-dexize round-trip" begin
+        @test juliaize(dexize(Int64(3))) === Int64(3)
+        @test juliaize(dexize(Int32(3))) === Int32(3)
+        @test juliaize(dexize(UInt8(3))) === UInt8(3)
+        @test juliaize(dexize(Float64(3))) === Float64(3)
+        @test juliaize(dexize(Float32(3))) === Float32(3)
+        @test juliaize(dexize(UInt64(3))) === UInt64(3)
+        @test juliaize(dexize(UInt32(3))) === UInt32(3)
```

Comment:
no need to change anything in this PR, but I'm just curious: does Julia have a package for property based testing? This whole test ultimately says that `juliaize . dexize == id`.

Reply:
It does not.
Would be great.